core/src/main/java/com/csun/game/UI/impl/settings/PauseUI.java

```diff
@@ -0,0 +1,161 @@
+
+        buttonTable.center();
+        buttonTable.add(resumeButton).width(150f).height(60f).pad(10f).row();
+        buttonTable.add(settingsButton).width(150f).height(60f).pad(10f).row();
+        buttonTable.add(exitButton).width(150f).height(60f).pad(10f).row();
+
+        stage.addActor(mainTable);
+        mainTable.setFillParent(true);
+
+        addActionsToButtons(player);
+    }
+
+    private void InitializeUIElements(){
+        TextureAtlas uiTextureAtlas = assetManager.get("atlas/ui/ui.atlas");
+        mainTable = new Table();
+
+        windowTable = new Table();
+        windowTable.background(new TextureRegionDrawable(uiTextureAtlas.findRegion("window")));
+
+        buttonTable = new Table();
+        iconTable = new Table();
+        ImageTextButton.ImageTextButtonStyle buttonStyle = new ImageTextButton.ImageTextButtonStyle(
+            new TextureRegionDrawable(uiTextureAtlas.findRegion("button")),null,null,
+            createFont(32));
+
+        resumeButton = new ImageTextButton("Resume",buttonStyle);
+        settingsButton = new ImageTextButton("Settings",buttonStyle);
+        exitButton = new ImageTextButton("Exit",buttonStyle);
+
+        ripperIcon = new Image(uiTextureAtlas.findRegion("icon-ripper"));
+    }
+    private void addActionsToButtons(Player player){
+        resumeButton.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                super.clicked(event, x, y);
+                List<BaseUI> openedUi = player.getAttributes().get(Attributes.ALL_OPENED_UI);
+                PauseUI.this.exit();
+                openedUi.remove(PauseUI.this);
+
+                player.getAttributes().set(Attributes.PLAYER_IS_PAUSED,false);
+            }
+        });
+        settingsButton.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                super.clicked(event, x, y);
+                //TODO show settingsUI
+            }
+        });
+        exitButton.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                super.clicked(event, x, y);
+                List<BaseUI> openedUi = player.getAttributes().get(Attributes.ALL_OPENED_UI);
+                PauseUI.this.exit();
+                openedUi.remove(PauseUI.this);
+
+                mainGame.switchScreen(MainGame.ScreenType.MENU_SCREEN);
+            }
+        });
+    }
+
+    @Override
+    public void loop(Player player, float deltaTime) {
+
+    }
+
+    @Override
+    public void exit() {
+        super.exit();
+        stage.clear();
+        mainGame.getInjector().getInstance(GameScreen.class).resume();
+    }
+
+    @Override
+    public boolean isWalkable() {
+        return false;
+    }
+
+
+    private BitmapFont createFont(int fontSize){
```

Comment:
Make this font a static final and initialize it in a static {} block so you don't need to create a new font every time you open the interface since the fonts don't change